### PR TITLE
Adding an MPI helper page

### LINKED ISSUE #15707: Improve MPI Instructions (I mean create them)
## Reason
Apparently we are not explaining how to use MOOSE with MPI.

## Design
Add some MPI instructions to our Getting Started instructions

## Impact
Show how to use MOOSE based apps with MPI

### PR DESCRIPTION
**Pull request by milljm**
Initial commit of an MPI markdown helper page.

Closes #15707

### CODE REVIEW

**Comment by cticenhour** [comment]
Reopening, since this got hit by the stale bot during the CIVET outage.

**Comment by aeslaughter** [comment]
@crswong888 Can you add this sort of information into one of the steps, if you have not already? Perhaps it's own step?

File: modules/doc/content/help/development/mpi.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+```pre
+Parallelism:
+  Num Processors:          2
+  Num Threads:             1
+```
+
+What's this about threads? Think of threads as 'virtual cores'. Every core has two threads. However, MPI does not make use of them. Threads is something an application must be willing to use. MOOSE, can make use of threads, but generally supplying physical cores is the better solution when solving large problems. Regardless, you can attempt to use a combination of multiple cores/threads with MOOSE by supplying the --n-threads=*num* argument:
+
+*Assuming you still residing in `~/projects/YourAppName/test/tests/kernels/simple_diffusion`*
```
Comment (milljm) [inline modules/doc/content/help/development/mpi.md:49]: you're

**Comment by milljm** [comment]
Need to add a section on how to make a simple MPI Hello World app. Since we seem to ask folks to do that all the time on the mailing list.

**Comment by crswong888** [comment]
@aeslaughter sure! I will find some way to incorporate this information.

**Comment by milljm** [comment]
@crswong888  was this ever included somewheres? I am being lazy and asking, before checking our website... sorry...

**Comment by crswong888** [comment]
@milljm yes we made it step 7 of the tutorial: [https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial01_app_development/step07_parallel.html](https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial01_app_development/step07_parallel.html)

**Comment by crswong888** [comment]
maybe it ought to be its own stand alone page that is more easy to find though...

**Comment by milljm** [comment]
Hmmm maybe. But having _something_ was way better than not! I don't know when the "TIPS" annotation went in. But those look good! Thanks!

**Comment by crswong888** [comment]
@milljm they went in to support the tutorial pages. Their ever-so subtly unique from "Note" alerts.

Let me know if you need anything else.

**Comment by aeslaughter** [comment]
I think adding something to the FAQ that links to this page would be a nice addition. I don't think we need to go too far, we haven't had anything for many, many years and people seemed to figure it out without much trouble. So, this addition will help a lot. 

@crswong888 Can you add something to the FAQ along with your next PR, no need to do something separate unless you want. We can close this then.

**Comment by crswong888** [comment]
@aeslaughter The FAQ link sounds sufficient. I got this on my to-do for my next PR

**Comment by aeslaughter** [comment]
You can close this when you add the FAQ bit. Thanks.

**Comment by crswong888** [comment]
I haven't forgot about this. Just been working on a MooseDocs PR and this issue doesn't fit well in it.

**Comment by crswong888** [comment]
I think this should have already been closed with #16639.

**Comment by aeslaughter** [comment]
closed with #16639.

**Pull request closed without merging**